src/eip7594/eip7594.c

```diff
@@ -505,23 +505,39 @@ static C_KZG_RET compute_commitment_to_aggregated_interpolation_poly(
         }
     }
 
-    /* Scale each cell with the corresponding powers of the random challenge */
-    for (uint64_t i = 0; i < num_cells; i++) {
-        for (size_t j = 0; j < FIELD_ELEMENTS_PER_CELL; j++) {
+    /*
+     * Vertically collapse cells of the 2D matrix into a single array: `aggregated_column_cells`.
+     *
+     * For each provided cell, go over its field elements, and scale them by the appropriate
+     * power of r.
+     *
+     * Then aggregate all field elements on the same vertical slice into a single array.
```

Comment:
Can we join these sentences into a single paragraph?

Reply:
Took the middle road in 9f6d28f3. Felt it looked better as a headline and a paragraph, but can merge the two if you prefer.

Reply:
Yup that's fine. Thank you.